beeline/__init__.py

```diff
@@ -559,15 +560,12 @@ def my_func(n):
         with this id.
     '''
 
-    _beeline = get_beeline()
-    if not _beeline:
-        # just pass through if not initialized
-        def wrapped(fn, *args, **kwargs):
-            @functools.wraps(fn)
-            def inner(*args, **kwargs):
+    def wrapped(fn, *args, **kwargs):
+        @functools.wraps(fn)
+        def inner(*args, **kwargs):
+            with tracer(name=name, trace_id=trace_id, parent_id=parent_id):
```

Comment:
is it valid to access `name`/`trace_id`/`parent_id` on the rhs here, or do they need to come through `kwargs` ala `kwargs["trace_id"]`?

Reply:
the args/kwargs is for the wrapped function, which has its own args. name, trace_id, and parent_id are in scope within the `traced` function that builds the wrapper.

Reply:
ah I see.  didn't look at the enclosing function definition 👍